### DISCUSSION TITLE
Why getIndicesFromDefaultFilters returns an array with zeroth index when expected is empty?

### DISCUSSION THREAD

**Discussion by pranavirabatti** [Sunbird-Knowlg/Community#17] [Q&A] [answered]
Why select the first element from the dropdown even when you deselect all the options from the dropdown?

![image](https://user-images.githubusercontent.com/22250006/172377237-5c9629a5-1f59-4c76-8aff-fb620b66cc39.png)

Expected is:

se_gradeLevels: [ ]
se_mediums: [ ]

@amit-tarento @vinukumar-vs please help us here

cc: @Renuka-S @komalm

**Comment by pramodkvarma** [comment, marked as answer]
@vinukumar-vs if you believe this needs code fix, do help @pranavirabatti to make a PR. That's the fastest way to get these done.

**Reply by vinukumar-vs** [reply]
@pranavirabatti This is the issue on Sunbird-ED portal. I would suggest to move this discussion to [Sunbrid-ed](https://github.com/Sunbird-Ed/Community/discussions)

@sharathkashyap, @luckynvdu Please help to resolve.

@Krishnaj20
